virttest/qemu_vm.py

```diff
@@ -5776,10 +5776,19 @@ def loadvm(self, tag_name):
         Override BaseVM loadvm method
         """
         self.verify_status("paused")  # Throws exception if not
-        LOG.debug("Loading VM %s from %s" % (self.name, tag_name))
+        LOG.debug("Loading snapshot %s from VM %s", tag_name, self.name)
         self.monitor.send_args_cmd("loadvm id=%s" % tag_name)
         self.verify_status("paused")  # Throws exception if not
 
+    def delvm(self, tag_name):
+        """
+        Override BaseVM delvm method
```

Comment:
According to https://github.com/avocado-framework/avocado-vt/blob/017ae5cc073e8d8110300d43eafa00dfe370c631/virttest/virt_vm.py#L1789-L1804
The `BaseVM` class just has the `savevm` and `loadvm` methods, not the delvm.
For this docsting, I suggest that we could update it without the `Override` string, or we can add the abstract method in the `BaseVM`.
Both approaches are okay for me, thanks.